de-lib/src/main/java/org/iplantc/de/desktop/client/presenter/DesktopPresenterWindowEventHandler.java

```diff
@@ -219,7 +221,7 @@ public void onRequestUploadFromUrl(RequestImportFromUrlEvent event) {
 
     @Override
     public void onRunAppActionInitiated(RunAppEvent event) {
-        AppWizardConfig config = ConfigFactory.appWizardConfig(event.getAppToRun().getId());
+        AppWizardConfig config = ConfigFactory.appWizardConfig(event.getAppToRun());
```

Comment:
It was. I added an overloaded version of `ConfigFactory.appWizardConfig` that takes an implementation of `HasQualifiedId`, which is implemented by `App`.